src/v0/destinations/kustomer/util.js

```diff
@@ -104,7 +104,7 @@ const handleResponse = (response) => {
       if (data && data.data && data.data.id) {
         return {
           userExists: true,
-          targetUrl: `${BASE_ENDPOINT}/v1/customers/${data.data.id}?replace=false`,
+          targetUrl: `${DEFAULT_BASE_ENDPOINT}/v1/customers/${data.data.id}?replace=false`,
```

Comment:
can you explain this change

Reply:
Yes, this was done to conform with the mocking existing for testcases earlier.

Reply:
`BASE_ENDPOINT` was being used to verify response for mocked testcases. so changed it to `DEFAULT_BASE_ENDPOINT as per responses

Reply:
what is purpose of target url, is it only for testing

Reply:
Yes, testing only